src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -163,9 +163,15 @@ where
         }
     }
 
-    fn process_startup_msg(&mut self, _msg: FeStartupMessage) -> Result<()> {
+    fn process_startup_msg(&mut self, msg: FeStartupMessage) -> Result<()> {
         // TODO: Replace `DEFAULT_DATABASE_NAME` with true database name in `FeStartupMessage`.
-        self.session = Some(self.session_mgr.connect("dev").map_err(IoError::other)?);
+        let db_name = {
+            match msg.config.get("database") {
+                None => "dev".to_string(),
+                Some(v) => v.to_string(),
+            }
+        };
```

Comment:
I think the default database name is "dev" whatever the Unix user name you used?

Reply:
> Pgsql will add dabatase name with unix user name if user not input the db name.

Yes, but it looks not so "cloud-native" to me. I think we will create a user for every cloud user. But for local development, +1 for `dev`.